standard-interfaces/cloud.astarte.GenericSensors.json

```diff
@@ -0,0 +1,13 @@
+{
+    "interface_name": "cloud.astarte.GenericSensors",
```

Comment:
This should be strongly typed, and we should have a set. E.g.: GenericDoubleSensors

Otherwise (which I believe could be even better), we should have several strongly typed endpoints (e.g.: /doubleValue, /intValue)

Reply:
At the end of the day sensors measure real values, and every integer or long integer can be safely converted to a double (with a minor loss of precision).
Measures have also an amount of error/noise, so I think that we don't need too much precision.
Special requirements might be covered with new interfaces (or custom ones).